src/server/pps/cmds/cmds.go

```diff
@@ -301,6 +301,22 @@ Examples:
 		}),
 	}
 
+	killJob := &cobra.Command{
+		Use:   "kill-job kill-id",
+		Short: "Kill a job.",
+		Long:  "Kill a job.  The job will be stopped immediately.",
```

Comment:
Do we want to add here something along the lines of "A killed job is effectively the same as a failed job, but is manually terminated by a user." 

I'm not sure where the appropriate place to add these types of documentation details are, but `--help` seems pretty reasonable.